src/Microsoft.Identity.Client/Internal/Requests/AuthenticationRequestParameters.cs

```diff
@@ -84,7 +85,22 @@ internal class AuthenticationRequestParameters
 
         // TODO: ideally, this can come from the particular request instance and not be in RequestBase since it's not valid for all requests.
         public bool SendX5C { get; set; }
-        public string LoginHint { get; set; }
+        public string LoginHint
+        {
+            get
+            {
+                if (_loginHint == null && Account != null)
```

Comment:
With strings, it's better to use String.IsNullOrEmpty because an empty string is semantically equivalent to a null string.